keps/sig-api-machinery/555-server-side-apply/README.md

```diff
@@ -286,6 +287,67 @@ The conversion between the two and creating the diff was complex and would have
 
 - 12/2019 [#86083](https://github.com/kubernetes/kubernetes/pull/86083) implementing a poc for the described approach
 
+
+#### API Audit
+
+The `ManagedFields` fields of an object in the API audit log may not be very useful. We want to provide a mechanism, 
+so the cluster operator can opt in so that the managed fields can be omitted from the audit log.
+
+We propose the following changes to the `audit.k8s.io/Policy` API that provides the cluster operator with a more 
+granular way to control the omission of managed fields in audit log:
+```go
+type Policy struct {
+	// +optional
+	OmitManagedFields bool `json:"omitManagedFields,omitempty"`
+}
+
+type PolicyRule struct {
+	// +optional
+	OmitManagedFields *bool `json:"omitManagedFields,omitempty"`
+}
+```
+The above API changes will be introduced in `v1`, `v1beta1` and `v1alpha1` of `audit.k8s.io`
+
+A new field `OmitManagedFields` is added to both `Policy` and `PolicyRule` making the following possible:
+- `Policy.OmitManagedFields` sets the default policy for omitting managed fields globally.
+  - the default value is `false`, managed fields are not omitted, this retains the current behavior.
```

Comment:
Any plans for changing the default value in a future release since the fields may not be very useful? making it opt-out instead?

Reply:
not that i am aware of, primarily we didn't want to quietly drop managed fields from audit log. The level `RequestResponse` stipulates that the response object in its "entirety" is being audited, so if we are to drop any fields from the audit log then I think it should be an opt-in by the operator.